Index: linux-1f77a85486f77fa30e90636c15d15aa9b6fd3ffc/scripts/dtc/dtc-lexer.lex.c_shipped
===================================================================
--- linux-1f77a85486f77fa30e90636c15d15aa9b6fd3ffc.orig/scripts/dtc/dtc-lexer.lex.c_shipped
+++ linux-1f77a85486f77fa30e90636c15d15aa9b6fd3ffc/scripts/dtc/dtc-lexer.lex.c_shipped
@@ -631,7 +631,7 @@ char *yytext;
 #include "srcpos.h"
 #include "dtc-parser.tab.h"
 
-YYLTYPE yylloc;
+extern YYLTYPE yylloc;
 extern bool treesource_error;
 
 /* CAUTION: this will stop working if we ever use yyless() or yyunput() */
