Index: linux-1f77a85486f77fa30e90636c15d15aa9b6fd3ffc/arch/arm64/Makefile
===================================================================
--- linux-1f77a85486f77fa30e90636c15d15aa9b6fd3ffc.orig/arch/arm64/Makefile
+++ linux-1f77a85486f77fa30e90636c15d15aa9b6fd3ffc/arch/arm64/Makefile
@@ -41,6 +41,7 @@ KBUILD_CFLAGS	+= $(lseinstr)
 KBUILD_CFLAGS	+= -fno-pic
 KBUILD_CFLAGS	+= $(call cc-option, -mpc-relative-literal-loads)
 KBUILD_CFLAGS	+= -fno-asynchronous-unwind-tables
+KBUILD_CFLAGS	+= $(call cc-disable-warning, address-of-packed-member)
 KBUILD_AFLAGS	+= $(lseinstr)
 
 ifeq ($(CONFIG_CPU_BIG_ENDIAN), y)
