Index: linux-1f77a85486f77fa30e90636c15d15aa9b6fd3ffc/scripts/Makefile.extrawarn
===================================================================
--- linux-1f77a85486f77fa30e90636c15d15aa9b6fd3ffc.orig/scripts/Makefile.extrawarn
+++ linux-1f77a85486f77fa30e90636c15d15aa9b6fd3ffc/scripts/Makefile.extrawarn
@@ -11,6 +11,8 @@
 # ==========================================================================
 
 KBUILD_CFLAGS += $(call cc-disable-warning, packed-not-aligned)
+KBUILD_CFLAGS += $(call cc-disable-warning, stringop-truncation)
+KBUILD_CFLAGS += $(call cc-disable-warning, stringop-overflow)
 
 ifeq ("$(origin W)", "command line")
   export KBUILD_ENABLE_EXTRA_GCC_CHECKS := $(W)
@@ -28,6 +30,8 @@ warning-1 += $(call cc-option, -Wmissing
 warning-1 += $(call cc-option, -Wunused-but-set-variable)
 warning-1 += $(call cc-option, -Wunused-const-variable)
 warning-1 += $(call cc-option, -Wpacked-not-aligned)
+warning-1 += $(call cc-option, -Wstringop-truncation)
+warning-1 += $(call cc-option, -Wstringop-overflow)
 warning-1 += $(call cc-disable-warning, missing-field-initializers)
 
 warning-2 := -Waggregate-return
