Index: linux-a4e78ea509f7c32206d84301e674b9718583ace1/arch/arm64/boot/dts/rockchip/rk3568-rock-3-a.dts
===================================================================
--- linux-a4e78ea509f7c32206d84301e674b9718583ace1.orig/arch/arm64/boot/dts/rockchip/rk3568-rock-3-a.dts
+++ linux-a4e78ea509f7c32206d84301e674b9718583ace1/arch/arm64/boot/dts/rockchip/rk3568-rock-3-a.dts
@@ -17,7 +17,7 @@
 		rockchip,wake-irq = <0>;
 		/* If enable uart uses irq instead of fiq */
 		rockchip,irq-mode-enable = <1>;
-		rockchip,baudrate = <1500000>;  /* Only 115200 and 1500000 */
+		rockchip,baudrate = <115200>;  /* Only 115200 and 1500000 */
 		interrupts = <GIC_SPI 252 IRQ_TYPE_LEVEL_LOW>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&uart2m0_xfer>;
@@ -69,7 +69,7 @@
 		simple-audio-card,format = "i2s";
 		simple-audio-card,mclk-fs = <128>;
 		simple-audio-card,name = "rockchip,hdmi";
-		status = "okay";
+		status = "disabled";
 
 		simple-audio-card,cpu {
 				sound-dai = <&i2s0_8ch>;
@@ -79,6 +79,37 @@
 		};
 	};
 
+	sound {
+		status = "okay";
+		compatible = "simple-audio-card";
+		simple-audio-card,name = "test";
+		simple-audio-card,format="i2s";
+
+		simple-audio-card,bitclock-master = <&cpu_dai>;
+		simple-audio-card,frame-master = <&cpu_dai>;
+		simple-audio-card,mclk-fs = <256>;
+
+		cpu_dai: simple-audio-card,cpu {
+			sound-dai = <&i2s3_2ch>;
+		};
+
+		codec_dai: simple-audio-card,codec {
+			sound-dai = <&i2s3_out>;
+		};
+
+	};
+
+	i2s3_out: i2s3-out {
+		#sound-dai-cells = <0>;
+		compatible = "flatmax,bare";
+		status = "okay";
+	};
+
+
+	i2s3_2ch {
+		status = "okay";
+	};
+
 	pdmics: dummy-codec {
 		status = "disabled";
 		compatible = "rockchip,dummy-codec";
@@ -112,7 +143,7 @@
 	};
 
 	rk809_sound: rk809-sound {
-		status = "okay";
+		status = "disabled";
 		compatible = "simple-audio-card";
 		simple-audio-card,format = "i2s";
 		simple-audio-card,name = "rockchip,rk809-codec";
@@ -629,7 +660,7 @@
 };
 
 &i2c5 {
-	status = "okay";	
+	status = "okay";
 	pinctrl-0 = <&i2c5m0_xfer>;
 };
 
@@ -917,7 +948,7 @@
 	reset-gpios = <&gpio2 RK_PD6 GPIO_ACTIVE_HIGH>;
 	vpcie3v3-supply = <&pcie30_3v3>;
 	//num-lanes = <2>;
-	pinctrl-0 = <&pcie30x2m1_pins>; 
+	pinctrl-0 = <&pcie30x2m1_pins>;
 	status = "okay";
 };
 
@@ -926,9 +957,9 @@
 };
 
 &pcie2x1 {
-	reset-gpios = <&gpio3 RK_PC1 GPIO_ACTIVE_HIGH>;	
+	reset-gpios = <&gpio3 RK_PC1 GPIO_ACTIVE_HIGH>;
 	vpcie3v3-supply = <&pcie30_3v3>;
-	pinctrl-0 = <&pcie20m1_pins>; 
+	pinctrl-0 = <&pcie20m1_pins>;
 	status = "okay";
 };
 
